python/HISTORY.rst

```diff
@@ -2,6 +2,13 @@
 History
 =======
 
+0.19.0 (2022-02-01)
+-------------------
+
+* Reads using "detached" commit-ids w/o specifying a branch or tag name
+* Support for Iceberg views (experimental)
```

Comment:
should we name this maybe `Replace SqlView support with Iceberg Views (experimental)`?

Reply:
Guess it's time - nobody used `SqlView` so far.